backend/apps/matching/configuration/properties.py

```diff
@@ -205,6 +220,18 @@ def generate_random_assignment(self, rs=None):
 
         return [rs.choice([True, False]) for i in range(len(self.choices))]
 
+    def get_filters(self) -> List:
+        return [
+            [
+                {
+                    "lookup_exp": "exact",
+                    "label": _("is exactly"),
+                    "model_field": models.NullBooleanField(blank=True, null=True),
+                }
+            ]
+            for i in range(len(self.choices))
```

Comment:
Is this the modern equivalent to a blank line?

Reply:
das ist der modern equivalent zu nicht required